### PR TITLE
Add voting sample 

### PR DESCRIPTION
**Pull request by jkotalik**
Fixes https://github.com/dotnet/tye/issues/197

@davidfowl what do I need to do to convert the ingress.yaml to the tye format?

### CODE REVIEW

File: samples/voting/tye.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  bindings:
+    - port: 5007
+- name: redis
+  dockerImage: redis
+  bindings:
+    - port: 6379
+- name: worker
+  project: worker/worker.csproj
+  bindings:
+    - port: 5000
```
Comment (davidfowl) [inline samples/voting/tye.yaml]: Is the worker HTTP based?

File: samples/voting/tye.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+name: VotingSample
```
Comment (davidfowl) [inline samples/voting/tye.yaml]: Remove all of the ports for individual projects. It should autogenerate ports.

File: samples/voting/tye.yaml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+services:
+- name: vote
```
Comment (davidfowl) [inline samples/voting/tye.yaml]:
```yml
ingress:
  rules: 
     - path: /vote
       service: vote
     - path: /results
       service: results
```

Comment (jkotalik) [reply samples/voting/tye.yaml]: Is this only for local for now?

**Pull request merged**